src/main/java/gov/cabinetoffice/gap/adminbackend/entities/SchemeEntity.java

```diff
@@ -53,6 +56,16 @@ public class SchemeEntity {
     @Column(name = "scheme_contact")
     private String email;
 
+    @ManyToMany(fetch = FetchType.LAZY, cascade = {CascadeType.MERGE, CascadeType.PERSIST})
+    @JoinTable(name = "scheme_permissions",
```

Comment:
Thinking that 'scheme_editors' might be better as its more consistent with our design i.e owner and editor. What do you think?

Reply:
yeah agreed - initially I had misunderstood and whacked a permission column on there, but yeah since removed that after Ryan corrected me!